packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -36,6 +36,17 @@ export interface SetCollateralToken {
   activateAfterBlock: number // ---| 4 bytes unsigned, Changes will be active after the block height
 }
 
+/**
+ * SetLoanToken DeFi Transaction
+ */
+export interface SetLoanToken {
+  symbol: string // ------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Symbol or id of collateral token
+  name: string // --------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Token's name, no longer than 128 characters
+  priceFeedId: string // -------| 32 bytes, hex string, Txid of oracle feeding the price
+  mintable: boolean // ---------| 4 bytes, mintable, Token's 'Mintable' property
```

Comment:
this is one byte in the tests.
it would be great if we could prepare https://github.com/DeFiCh/jellyfish/pull/568 PR before this.
we could extract data from it.